modules/auxiliary/admin/http/tomcat_ghostcat.rb

```diff
@@ -70,13 +70,16 @@ def make_headers(headers)
     headers_ajp = Array.new
     for (header_name, header_value) in headers do
       code = header2code[header_name].to_s
+
+      # rubocop:disable Style/IdenticalConditionalBranches
       if code != ''
         headers_ajp.append(code)
         headers_ajp.append(ajp_string(header_value.to_s))
       else
         headers_ajp.append(ajp_string(header_name.to_s))
         headers_ajp.append(ajp_string(header_value.to_s))
       end
+      # rubocop:enable Style/IdenticalConditionalBranches
```

Comment:
is this an issue with rubocop? the branches aren't identical?

Reply:
There's duplicate code that could be extracted. I believe it's meant to be this:

```
      if code == ''
        headers_ajp.append(code)
      else
        headers_ajp.append(ajp_string(header_name.to_s))
      end
      headers_ajp.append(ajp_string(header_value.to_s))
```

I don't want to start shuffling code around even if the fix is obvious

Reply:
Extra context:

https://docs.rubocop.org/rubocop/cops_style.html#styleidenticalconditionalbranches

Reply:
>The cop is poorly named and some people might think that it actually checks for duplicated conditional branches. The name will probably be changed in a future major RuboCop release.

hah, yea, no that makes perfect sense actually

Reply:
Marking this as resolved for now, however this is definitely something we can come back to in the future and this type of check is something I'd like to look into further.